assets/src/stories-editor/blocks/amp-story-page/edit.js

```diff
@@ -43,7 +43,7 @@ import {
 	getCallToActionBlock,
 	getUniqueId,
 	uploadVideoFrame,
-	getPosterImageFromFileObj,
+	getPosterImageFromFileObj, processMedia,
```

Comment:
^ This could be in a new line for consistency.

Reply:
I really need to find an ESLint rule at some point to catch these kind of things 🤔

Reply:
There was nothing in the linter for this. 
FIxed in https://github.com/ampproject/amp-wp/pull/3093/commits/62aa7fbdd2203f22a04ef77491dedafac3bfeb13